pkg/translator/prometheusremotewrite/metrics_to_prw_v2.go

```diff
@@ -80,7 +80,12 @@ func (c *prometheusConverterV2) fromMetrics(md pmetric.Metrics, settings Setting
 					}
 					c.addGaugeNumberDataPoints(dataPoints, resource, settings, promName)
 				case pmetric.MetricTypeSum:
-					// TODO implement
+					dataPoints := metric.Sum().DataPoints()
+					if dataPoints.Len() == 0 {
+						errs = multierr.Append(errs, fmt.Errorf("empty data points. %s is dropped", metric.Name()))
```

Comment:
does this need to be returned as an error?  Can we just ignore these points instead?  `addSumNumberDataPoints` won't do anything if there are no datapoints